web/impact/impact/tests/test_industry_detail_view.py

```diff
@@ -43,3 +45,13 @@ def test_options(self):
             assert response.status_code == 200
             get_options = response.data["actions"]["GET"]["properties"]
             assert_fields(INDUSTRY_GET_FIELDS, get_options)
+
+    def test_options_against_get(self):
```

Comment:
noting that the added tests don't fail on development. I guess this is in line with functionality not being added.

Reply:
This one doesn't, but test_options_against_get from test_user_confidential_view.py and test_program_family_detail_view.py do.  It depends on how well the old OPTIONS match the new GET response.  Won't fix.